pallets/pallet-dip-consumer/src/lib.rs

```diff
@@ -105,6 +104,9 @@ pub mod pallet {
 	/// The pallet contains a single storage element, the `IdentityEntries` map.
 	/// It maps from a subject `Identifier` to an instance of
 	/// `LocalIdentityInfo`.
+	// SAFETY of Twox64Concat:
+	// The Identifier cannot be chosen freely it is therefore not efficient to
+	// craft specific hashes.
```

Comment:
Same comment as for the provider pallet. Do you think it's a better design if the pallet ensures safety by not letting the runtime mess up? Technically, the `IdentityEntries` is populated using the information in the origin. In the case of DIDs this is probably not possible, but in the case of other IDs this could still be possible by providing the right origin.

Alghouth, we have not made the same considerations for the runtime's generic `AccountId` tho...